src/modules/0.0.18/aws_cloudfront/module.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "aws_cloudfront",
+    "dependencies": ["iasql_platform", "aws_account"]
```

Comment:
not related to this PR, but we might eventually have this depend on s3 so users can expose files from it?

Reply:
I even didn't add support for s3 yet... only custom distro. But we can add in next steps